AlphaWallet/Core/CoinTicker/Types/TickerIdFilter.swift

```diff
@@ -67,6 +67,7 @@ struct TickerIdFilter {
         case .binance_smart_chain: return platform == "binance-smart-chain"
         case .avalanche: return platform == "avalanche"
         case .polygon: return platform == "polygon-pos"
+        case .candle: return platform == "candle"
```

Comment:
I took a quick look, it doesn't look like candle is supported by CoinGecko, as in they don't provide data for tokens on Candle.

Would you help exclude this line too? We can add it again when there's data.

(CoinGecko does provide data for 0x3D97EdB1c1D87C0cBf098a0D2230d7380d4b1432 on Ethereum mainnet, that's fine, this line of code isn't about that, it's about tokens on Candle)

Reply:
Yes